.github/dependabot.yml

```diff
@@ -11,3 +11,5 @@ updates:
     labels:
       - "auto-approve"
     open-pull-requests-limit: 5
+    ignore:
```

Comment:
I'm pretty sure you need to add a new top-level entry with `package-ecosystem: pip`.

I think `package-ecosystem: github-actions` is intended to update the verisons of GitHub Actions you're using.

Reply:
Actually, I'm on the wrong repo, even!

Reply:
Haha, so you are